modules/recon/hosts-hosts/hackertarget_reverse.py

```diff
@@ -0,0 +1,36 @@
+from recon.core.module import BaseModule
+
+
+class Module(BaseModule):
+    meta = {
+        'name': 'HackerTarget reverse IP lookup',
+        'author': 'Qazeer',
+        'version': '1.0',
+        'description': 'Uses the HackerTarget.com reverse IP lookup API to find host names associated with IP addresses. Updates the \'hosts\' table with the results.',
+        'query': 'SELECT DISTINCT ip_address FROM hosts WHERE ip_address IS NOT NULL',
+    }
```

Comment:
Use the optional `comments` section here to leave a note about the API limitations.

Also, is there a way to handle the API limitations? For instance, if I have an API key, can I get more requests? If so, this module needs to support that. Please incorporate API key handling for the API if it is supported by HackerTarget.